code/Base/Hud/Chat/ChatBox.scss

```diff
@@ -0,0 +1,150 @@
+﻿SportsChatBox {
+	position: absolute;
+	bottom: 200px;
+	left: 100px;
+	z-index: 100;
+	flex-direction: column;
+	width: 500px;
+	font-family: Poppins;
+	font-size: 13px;
+
+	.party_chat {
+		background-color: black;
+		opacity: 0;
+		color: white;
+		padding: 15px;
+		background-color: rgba(black, 0.5);
+		font-size: 14px;
+		flex-shrink: 0;
+		flex-grow: 0;
+		width: 100%;
+	}
+
+	.chat_canvas {
+		flex-direction: column;
+		align-items: flex-start;
+	}
+
+	&.open {
+		.party_chat {
+			opacity: 1;
+			backdrop-filter: blur(10px);
+		}
+	}
+	&.global.open {
+		.globalChat {
+			display: flex;
+		}
+		.partyChat {
+			display: none;
+		}
+	}
+	&:not(.global).open {
+		.globalChat {
+			display: none;
+		}
+		.partyChat {
+			display: flex;
+		}
+	}
+	.partyChat,
+	.globalChat {
+		display: none;
+		width: 100px;
+		max-height: 100px;
+		font-size: 20px;
+		position: absolute;
+		font-family: Poppins;
+		color: white;
+		background-color: rgba(rgb(190, 112, 39), 0.5);
+
+		right: 0;
+		height: 100%;
+		text-align: center;
+		align-items: center;
+	}
+	.partyChat {
```

Comment:
Let's space our same-level style classes out by one line (like you did below) here.